Core/src/main/java/com/plotsquared/core/player/PlotPlayer.java

```diff
@@ -841,6 +842,27 @@ public void sendTitle(
                 .title(titleComponent, subtitleComponent, times));
     }
 
+    /**
+     * Sends an actionbar to the player on plot entry.
+     * This takes in two captions as it will internally merge
+     * the captions together to send as a single component.
```

Comment:
We might need to outline that this method is explicitly used for titles as actionbars. Because you very likely can't/wont use this method for forming regular actionbars.

Reply:
So do you just want an updated explanation of it in the javadocs?

Reply:
Yeah. The current describing sounds like it can be used everywhere to send actionbars.

Reply:
Done!

Reply:
Thanks